radicals/urls.py

```diff
@@ -21,13 +21,13 @@
 
 urlpatterns = [
     path('admin/', admin.site.urls),
-    path('', lambda request: HttpResponseRedirect('home/')),
-    path('backoffice/customers/', include('back_office.customers.urls')),
-    path('backoffice/invoices/', include('back_office.invoices.urls')),
-    path('backoffice/orders/', include('back_office.orders.urls')),
-    path('backoffice/products/', include('back_office.products.urls')),
-    path('backoffice/sales/', include('back_office.sales.urls')),
-    path('backoffice/settings/', include('back_office.settings.urls')),
+    path('', lambda request: HttpResponseRedirect('/home/')),
+    path('customers/', include('back_office.customers.urls')),
```

Comment:
These should not be changed, change back to them having backoffice at the start as it was

Reply:
Fixed the urls.py and also found a issue with dropdown menu but it is solved now.

Reply:
lines like this should not exist:
path('settings/', include('back_office.settings.urls')),
if it has backoffice in the include function, the backoffice should be present in the path string, like this:
path('backoffice/customers/', include('back_office.customers.urls')),

Delete the wrong ones

Reply:
Now should be ok.